docs/querying/granularities.md

```diff
@@ -61,6 +61,8 @@ The minimum and maximum granularities are `none` and `all`, described as follows
 * `none` does not mean zero bucketing. It buckets data to millisecond granularity—the granularity of the internal index. You can think of `none` as equivalent to `millisecond`.
   > Do not use `none` in a [timeseries query](../querying/timeseriesquery.md); Druid fills empty interior time buckets with zeroes, meaning the output will contain results for every single millisecond in the requested interval.
 
+*Avoid using the `week` granularity string, because weeks don't align neatly with months and years, making it difficult to partition by coarser granularities later.
```

Comment:
```suggestion
*Avoid using the `week` granularity for partitioning at ingestion time, because weeks don't align neatly with months and years, making it difficult to partition by coarser granularities later.
```
This doc applies to both querying and ingestion, so suggest clarifying that's when to avoid. Not sure we need to repeat the `string` bit of `granularity string` here.